amazon/ion/core.py

```diff
@@ -113,7 +113,19 @@ def scale_to_precision(fractional_seconds, fractional_precision):
     Returns:
         Decimal: The scaled value.
     """
-    return Decimal(fractional_seconds) * Decimal(Decimal(10) ** Decimal(-fractional_precision))
+    return Decimal(fractional_seconds).scaleb(-fractional_precision)
+
+
+def round_to_sixth_precision(fractional_seconds):
```

Comment:
*Minor:* Prefix this with `_` to imply that it is not part of the public surface, this goes for the `scale_to_precision` method above as well (I missed that on first pass).

Also, `round_to_sixth_precision` is not a helpful name, perhaps `_fractional_seconds_to_microseconds`.

Reply:
Done. Changed name to `_fractional_seconds_to_microseconds` and prefixed `scale_to_precision` with `_`.